client/payment-details/payment-method/klarna/index.js

```diff
@@ -0,0 +1,123 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import PaymentDetailsPaymentMethodDetail from '../detail';
```

Comment:
Since `PaymentDetailsPaymentMethodDetail` is the default export you can import it as `Detail`. This way you don't need to re-assign it later.

```suggestion
import Detail from '../detail';
```